specs/analytics_v1_pageviews.yaml

```diff
@@ -0,0 +1,291 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: Analytics Pageviews API
+  description: Analytics Pageviews API.
+  termsofservice: https://github.com/wikimedia/restbase#restbase
+  contact:
+    name: Analytics
+    email: analytics@lists.wikimedia.org
+    url: https://www.mediawiki.org/wiki/Analytics
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+x-default-params:
+  domain: analytics.wikimedia.org
```

Comment:
This is not needed here. It's used in monitoring to pre-set the default domain in a monitor. Since you are not using monitoring (`x-monitor: false`) this is not needed.